src/Pages/Payment/Payment.component.js

```diff
@@ -0,0 +1,226 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import { connect } from "react-redux";
+
+import OrderProductList from "./OrderProductList/OrderProductList.component";
+import PaymentButton from "./PaymentButton/PaymentButton.component";
+
+import "./Payment.styles.scss";
+
+import PaymentMethod from "./PaymentMethod/PaymentMethod.component";
+import { getCartItems } from "../../redux/cart/cart.actions";
+
+import { GET_SHOPPINGBASKET_API } from "../../config";
+
+class Payment extends Component {
+  constructor() {
+    super();
+    this.state = {
+      onOffSwitch: true,
+      checkPayMethod: "",
+    };
+  }
+
+  numberWithCommas = (price) =>
+    price.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+
+  handleCheckBox = (e) => {
+    this.setState({ checkPayMethod: e.target.name });
+  };
+
+  componentDidMount = async () => {
+    const { userToken, getCartItems } = this.props;
+    await fetch(GET_SHOPPINGBASKET_API, {
+      headers: {
+        "content-type": "application/json",
+        Authorization: userToken,
+      },
+    })
+      .then((res) => res.json())
+      .then((data) => data.shopping_list)
+      .then((cartItems) => getCartItems(cartItems))
+      // .then((data) => console.log(data))
+      .catch((error) => console.log(error));
+  };
+
+  render() {
+    const { onOffSwitch } = this.state;
+    const { cartItems, currentUser } = this.props;
+    const orderingItems = cartItems.filter((item) => item.checked === true);
+    const totalAmount = orderingItems.reduce(
+      (acc, curVal) => acc + curVal.discount_price * curVal.quantity,
+      0
+    );
+    console.log(cartItems);
+    console.log(orderingItems);
+    return (
+      <div className="Payment">
+        <div className="payment-container">
+          <header>
+            <h2>주문서</h2>
+            <p>주문하실 상품명 및 수량을 정확하게 확인해 주세요.</p>
+          </header>
+          <section className="section-name product-info">
+            <h3>상품 정보</h3>
+            <div className="product-info-container">
+              <div
+                className={`products-list-before ${onOffSwitch ? "on" : "off"}`}
+              >
+                <p>{`${orderingItems && orderingItems[0].name} 외 ${
+                  orderingItems && orderingItems.length - 1
+                }종`}</p>
```

Comment:
- orderingItems는 배열이므로 boolean 값을 체크했을 경우 언제나 true가 리턴됩니다. 따라서 현재 의미없는 조건문입니다. 만약 체크한다면 .length를 확인해주세요.
- 여기에 해당되는 내용은 아니지만 옵셔널 체이닝을 사용하면 보다 간편하게 falsy한 값을 체크할 수 있습니다.
```suggestion
                <p>{`${orderingItems?.[0].name} 외 ${orderingItems?.length - 1}종`}</p>
```